install_custom_module_map/after/Pods/Target Support Files/Pods-Example/Pods-Example-frameworks.sh

```diff
@@ -44,6 +44,16 @@ install_framework()
     source="$(readlink "${source}")"
   fi
 
+  if [ -d "${source}/${BCSYMBOLMAP_DIR}" ]; then
+    # Locate and install any .bcsymbolmaps if present, and remove them from the .framework before the framework is copied
+    find "${source}/${BCSYMBOLMAP_DIR}" -name "*.bcsymbolmap"|while read f; do
```

Comment:
do you think if we add `-type f` in the `find` invocation things can be a bit faster?

Reply:
potentially - these folders tend to only have 1-2 files in them so I doubt it would make a huge difference